lib/librato/metrics/smart_json.rb

```diff
@@ -1,19 +1,27 @@
-require 'multi_json'
+USE_MULTI_JSON = defined?(MultiJson)
+
+require 'json' unless USE_MULTI_JSON
+require 'multi_json' if USE_MULTI_JSON
 
 module Librato
   module Metrics
     class SmartJSON
-      JSON_HANDLER = MultiJson
+      JSON_HANDLER =
+        if USE_MULTI_JSON
+          MultiJson
+        else
+          JSON # ships with 1.9
+        end
       extend SingleForwardable
-      
+
       # wrap MultiJSON's implementation so we can use any version
       # prefer modern syntax if available; def once at startup
       if JSON_HANDLER.respond_to?(:load)
```

Comment:
This unfortunately now has to be more complicated, because if we are using stdlib json we want to use #parse (because of [this security aspect](http://ruby-doc.org/stdlib-2.0.0/libdoc/json/rdoc/JSON.html#method-i-load)). So probably you need to do something like (psuedo-code):

```rb
if defined?(MultiJson)
  .. do current logic
else
  require 'json'
  ..wire up json delegators
end
```